docs/versions

```diff
@@ -1,4 +1,3 @@
 master              master
 release/v0.44.x     v0.44
 release/v0.45.x     v0.45
-
```

Comment:
I find that diff not obvious tbh, as a result, I thought this line was missing in a previous pr.